### ISSUE TITLE
Compiling Receiver on Fedora 34 with pipewire

### ISSUE THREAD

**Issue by ShrirajHegde** [duncanthrax/scream#141]
Receiver doesn't seem to work on fedora 34 with pipewire. Is there any way to get it working? if yes, how?

**Comment by duncanthrax** [comment]
I have never seen pipewire, but I would guess it is runtime-compatible with pulseaudio. Meaning, you'll have to have pulseaudio development packages to build the receiver, but you can then run the receiver against pipewire.

**Comment by ShrirajHegde** [comment]
yeah, pulse audio-dependent programs are supposed to be compatible with pipe wire, I have installed the guest driver. When I run the client on the host machine, I hear no audio, I have tried providing `virbr0`, `virbr1` as bridges for NAT. 

When run on the terminal, I get :

        Using ALSA/PulseAudio output
        Starting multicast receiver

but same when run as systemd service gives error `snd_pcm_open: Host is down.

I saw people using it on youtube and it worked right after installing the driver without any extra effort. I am yet to try it on ubuntu(with same VM) and then I can confirm

**Comment by duncanthrax** [comment]
> snd_pcm_open

This looks like something can't open ALSA.

**Comment by acuteaura** [comment]
Running into similar issues on fc34.

```
$ ./scream -o alsa -i virbr0
snd_pcm_open: Host is down
```

```
$ ./scream -o pulse -i virbr0
Unable to connect to PulseAudio. Connection refused
```

```
$ sudo getenforce
Permissive
```

Have not really run into any other issues with pipewire.

**Comment by ShrirajHegde** [comment]
>Have not really run into any other issues with pipewire.

Same

**Comment by acuteaura** [comment]
My issue turned out to be a temporary fluke where somehow half my environment, including dbus session went missing.
